examples/getting-started-movielens/03-Training-with-PyTorch.ipynb

```diff
@@ -37,45 +37,36 @@
     "\n",
```

Comment:
<p>Can we not use the Schema file instead of CATEGORICAL_COLUMNS and CATEGORICAL_MH_COLUMNS and NUMERIC_COLUMNS parameters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>